docs/source/en/features/pipeline_parallel.md

```diff
@@ -60,101 +59,155 @@ In this schedule, each device can perform computation for multiple subsets of la
 
 This mode is both memory-efficient and time-efficient.
 
-## Usage of non-interleaved and interleaved schedule
+## Colossal-AI's Implement
```

Comment:
Should be `Implementation` rather than `Implement`. `Implement` means "tools" in noun.